documentation/specs/DisplayTable.md

```diff
@@ -0,0 +1,238 @@
+# `DisplayTable` Component Specification
```

Comment:
tried to name this to be agnostic of its location. in theory it could be used in our products and so i didn't want the name to limit us in that regard. open to better names

Reply:
Looked at a few different systems for inspiration and seeing `Data Grid` come up as a name for what we are naming `Table` ("interactive tabular interface") and `Table` for this component ("display simple datasets that do not need to be filtered or edited"). Don't feel super strongly but I do like Data Grid for the other table we're building.
Goldman Sachs: [Data Grid](https://design.gs.com/components/data-grid)/[Table](https://design.gs.com/components/table)
Edison™ Design System: [Data Grid](https://edisondesignsystem.com/datagrid)/[Table](https://edisondesignsystem.com/table?tab=0)

Reply:
i like that. we'll use that scheme